pkgs/top-level/all-packages.nix

```diff
@@ -38062,4 +38062,6 @@ with pkgs;
   aitrack = libsForQt5.callPackage ../applications/misc/aitrack { };
 
   widevine-cdm = callPackage ../applications/networking/browsers/misc/widevine-cdm.nix { };
+
+  asusctl = callPackage ../applications/system/asusctl { };
```

Comment:
Please sort this.

Reply:
Where shall I put it exactly ? I don't see the logic in the all-packages.nix as everything is not sorted.

Reply:
Try to find ``### APPLICATIONS`` or sort it near other applications but appending it provokes merge conflicts.